pom.xml

```diff
@@ -4,26 +4,27 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.11</version>
+    <version>3.2</version>
+    <relativePath />
   </parent>
 
   <artifactId>cobertura</artifactId>
   <packaging>hpi</packaging>
   <version>1.13-SNAPSHOT</version>
   <name>Jenkins Cobertura Plugin</name>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/Cobertura+Plugin</url>
+  <url>https://wiki.jenkins.io/display/JENKINS/Cobertura+Plugin</url>
 
   <properties>
     <jenkins.version>2.13</jenkins.version>
```

Comment:
I would bump it to 2.19.3 so that tests run for LTS

Reply:
Agreed, but leaving that to the plugin maintainer to do later if he agrees.

Reply:
Makes sense to me, but why not 2.19.4?

Reply:
2.19.4 works as well. I just forgot we have .4 in that LTS line